webpush_channels/tests/test_views_channel.py

```diff
@@ -174,3 +176,37 @@ def test_post_request_response_is_json(self):
         resp = self.app.post(self.channel_url, headers=self.headers, status=202)
         assert resp.body == b'{}'
         assert resp.headers['Content-Type'] == 'application/json'
+
+    def test_put_request_on_invalid_channel_registration_url_response_is_json(self):
+        resp = self.app.put(self.channel_url, headers=self.headers, status=405)
+        assert self.resp.body == '{}'
+        assert self.resp.headers['Content-Type'] == 'application/json'
+
+    def test_delete_request_on_invalid_registration_url_response_is_json(self):
+        resp = self.app.delete(self.channel_url, headers=self.headers, status=405)
+        assert json.loads(resp.body)
+        assert resp.headers['Content-Type'] == 'application/json'
+
+    def test_delete_request_on_non_existent_registration_url_response_is_json(self):
+        resp = self.app.delete(self.invalid_channel_registration_url,
+                               headers=self.headers,
+                               status=202)
+        assert resp.body == '{}'
+        assert resp.headers['Content-Type'] == 'application/json'
+
+    def test_get_request_on_non_existent_channel_url_response_is_json(self):
+        resp = self.app.get(self.invalid_channel_url, headers=self.headers, status=403)
+        assert resp.body == '{}'
```

Comment:
You can use this mixing to validate the errors: https://github.com/Kinto/kinto/blob/master/kinto/core/testing.py#L68